### DISCUSSION TITLE
I want to participate in GSoC 2023, need help to get started with open source.

### DISCUSSION THREAD

**Discussion by dmdbilal** [openfoodfacts/openfoodfacts-androidapp#4900] [Q&A]
Hey everyone, I am Mohamed Bilal , second year B.Tech student at Sri Sairam engineering college , Chennai, India. I know Android development related stuffs : Kotlin , MVVM, Room, APIs , Retrofit etc.. I need guidance to start working on this project.So, can you tell me more about this and guide me further about the same.

**Comment by rishujam** [comment]
will open food facts participate in GSOC 2023 ?

**Reply by dmdbilal** [reply]
@rishujam I don't know.
